controllers/sms-gateway.js

```diff
@@ -73,9 +80,11 @@ function getOutgoing(callback) {
         content: message.message,
       };
     });
-    markMessagesForwarded(messages, function() {
+    // TODO add this back in once this function also queries for
+    // messages in state: 'forwarded-to-gateway'
+    // markMessagesForwarded(messages, function() {
```

Comment:
Gateway calls the sms-gateway API which uses the same code in message-utils.js... I'm thinking it's easy to give the gateway messages in both `pending` and `forwarded-to-gateway` states if it needs them.